features/steps/python_s2i.py

```diff
@@ -45,8 +45,9 @@ def step_impl(context, s2i: str):
             assert entry.get("thoth_s2i_image_version"), f"No version identifier found for {s2i!r} Thoth s2i"
             assert entry.get("analysis_id"), f"No container image analysis found for {s2i!r} Thoth s2i"
             thoth_s2i = f"{s2i}:{entry['thoth_s2i_image_version']}"
```

Comment:
Would something like this work?

```suggestion
            thoth_s2i = f"{s2i}:v{entry['thoth_s2i_image_version']}"
```

Reply:
yes that would work as well.
i had made it in a current way, such that suppose someone released a tag without `v`, then it would cause issue
maybe I over-engineered this, completely forgot that we define the packages version on release

Reply:
fixed as you suggested :+1: